src/PlusCommon/Testing/CMakeLists.txt

```diff
@@ -97,39 +97,37 @@ IF(PLUSBUILD_BUILD_PlusLib_TOOLS)
     SegmentationTest_BKMedical_RandomStepperMotionData2_Trimmed.mha)
 
   #--------------------------------------------------------------------------------------------
+  IF(VTK_VERSION_MAJOR LESS 9)

```

Comment:
In latest VTK, major version has been reverted to 8 (instead of 9.0 now master is 8.2), so it would be safer to check version >= 8.2